wgengine/router/router_linux.go

```diff
@@ -1285,7 +1148,7 @@ func (r *linuxRouter) addNetfilterBase() error {
 	if err := r.addNetfilterBase4(); err != nil {
```

Comment:
In order rules, the difference between 4 and 6 is handled inside the netfilter runner. Can the runner interface drop AddNetfilterBase4 and AddNetfilterBase6, and just have AddNetfilterBase?

Reply:
Yeah we totally can.